plugin-jdbc-clickhouse/src/main/java/io/kestra/plugin/jdbc/clickhouse/cli/ClickHouseCLI.java

```diff
@@ -0,0 +1,120 @@
+package io.kestra.plugin.jdbc.clickhouse.cli;
+
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.tasks.*;
+import io.kestra.core.models.tasks.runners.ScriptService;
+import io.kestra.core.models.tasks.runners.TaskRunner;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.scripts.exec.scripts.models.ScriptOutput;
+import io.kestra.plugin.scripts.exec.scripts.runners.CommandsWrapper;
+import io.kestra.plugin.scripts.runner.docker.Docker;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotEmpty;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static io.kestra.core.utils.Rethrow.throwFunction;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+	title = "Runs a ClickHouse-local commands."
+)
+@Plugin(
+	examples = {
+		@Example(
+			title = "Query data in a Parquet file in AWS S3",
+            full = true,
+            code = {
+                """
+                id: clickhouse-local
+                namespace: company.team
+                tasks:
+                  - id: wdir
+                    type: io.kestra.plugin.core.flow.WorkingDirectory
+                    tasks:
+                      - id: query
+                        type: io.kestra.plugin.clickhouse.cli.ClickHouseCLI
+                        commands:
+                          - SELECT count() FROM s3('https://datasets-documentation.s3.eu-west-3.amazonaws.com/house_parquet/house_0.parquet')
+                """
+            }
+		)
+	}
+)
+public class ClickHouseCLI extends Task implements RunnableTask<ScriptOutput>, NamespaceFilesInterface, InputFilesInterface, OutputFilesInterface {
```

Comment:
I'll rename the task to just CLI in the clickhouse package:

io.kestra.plugin.jdbc.clickhouse.CLI